src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -12,7 +12,9 @@ define([
         interval: 30000,
         once: false,
         startatonce: true,
+        callEvent: null,
         microflow: "",
+        nanoflow: "",
```

Comment:
Nanoflow is not a string, it should be `nanoflow: null`

Reply:
done